libs/gpg45-evaluator/src/test/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ProfileEvaluatorTest.java

```diff
@@ -76,6 +82,25 @@ void buildScoreShouldReturnCorrectScoreForPassportCredential() throws Exception
         assertEquals(expectedScores, builtScores);
     }
 
+    @Test
+    void buildScoreShouldReturnCorrectScoreForDuplicateDrivingPermitCredential() throws Exception {
```

Comment:
You could consider adding another test case that covers a strength/validity + separate verification for the same doc, but fine as-is